content/How to Make My First Gem - Timothy Peraza.md

```diff
@@ -0,0 +1,128 @@
+# **How to make your first Ruby Gem (step-by-step)**
+
+Every developer who has worked with ruby on rails and has felt passion or interest for the open source world, has wanted or wants to do a contribution to this community of development with a gem which could solve a particular or generic problem or improve an existing one, that appears in different projects across the developer career.
+
+However, sometimes it happens that one doesn't know where to start or how is a gem made in the first place, or other times one thinks or has the idea that to make a gem is a really hard or complicated task. For that reason, this article is aimed to inquire how to make your first gem and to explain a bit the different components of it.
+
+Here at Rootstrap, we are very interested in being reconized in the open source world. For that reason, we have been working in this topic for some time, making libraries for diferents languages and frameworks and making them visible with blogs and posts in social media. In particular, we made [rsgem](https://github.com/rootstrap/rsgem), a gem which generates a base project to make a gem, in which one this article is based on.
+
+
+## Starting the Gem
+
+The first step to start is to install the rsgem, this is able to do easily by running `$ gem install rsgem`. After we have the gem installed, we have to run `$ rsgem new [name]`, with the name that we want for our gem. In particular, in this blog, we will be running the command `$ rsgem new blog_gem`, and this will generate a project with the following structure:
```

Comment:
```suggestion
The first step here is to install the rsgem by running `$ gem install rsgem`. Once the gem is installed, we have to run `$ rsgem new [name]`, with the name that we want for our gem included. In this blog, we will pay close attention we will be running the command `$ rsgem new blog_gem`, as this will generate a project with the following structure:
```